test/components/views/context_menus/MessageContextMenu-test.tsx

```diff
@@ -13,33 +13,205 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+/* eslint-disable @typescript-eslint/no-var-requires */
 
 import React from 'react';
-import { mount } from 'enzyme';
+import { mount, ReactWrapper } from 'enzyme';
 import { EventStatus, MatrixEvent } from 'matrix-js-sdk/src/models/event';
 import { Room } from 'matrix-js-sdk/src/models/room';
 import { PendingEventOrdering } from 'matrix-js-sdk/src/matrix';
 import { ExtensibleEvent, MessageEvent, M_POLL_KIND_DISCLOSED, PollStartEvent } from 'matrix-events-sdk';
+import { Thread } from "matrix-js-sdk/src/models/thread";
 
 import * as TestUtils from '../../../test-utils';
-import MessageContextMenu from '../../../../src/components/views/context_menus/MessageContextMenu';
 import { MatrixClientPeg } from '../../../../src/MatrixClientPeg';
+import RoomContext, { TimelineRenderingType } from "../../../../src/contexts/RoomContext";
+import { IRoomState } from "../../../../src/components/structures/RoomView";
+
+const PATH_TO_STRING_UTILS = "../../../../src/utils/strings";
+const PATH_TO_EVENT_UTILS = "../../../../src/utils/EventUtils";
+
+jest.mock(PATH_TO_STRING_UTILS);
+jest.mock(PATH_TO_EVENT_UTILS);
+
+const { copyPlaintext, getSelectedText } = require(PATH_TO_STRING_UTILS);
+const { canEditContent, canForward, isContentActionable } = require(PATH_TO_EVENT_UTILS);
+const MessageContextMenu = require("../../../../src/components/views/context_menus/MessageContextMenu")["default"];
```

Comment:
Importing this way loses all the types for `MessageContextMenu`, why not use an `import`?